lib/web/desktop.go

```diff
@@ -392,6 +392,33 @@ func proxyWebsocketConn(ws *websocket.Conn, wds net.Conn) error {
 	return trace.NewAggregate(retErrs...)
 }
 
+// wasWebsocketClosedByClient handles the error returned by proxyWebsocketConn
+func handleProxyWebsocketConnErr(err error, log *logrus.Entry) {
+	if err == nil {
+		return
+	}
+
+	if agg, ok := trace.Unwrap(err).(trace.Aggregate); ok {
+		wsClosedByClient := false
+		for _, err := range agg.Errors() {
+			if ce, ok := trace.Unwrap(err).(*websocket.CloseError); ok {
+				switch ce.Code {
+				case websocket.CloseNoStatusReceived, // when the user hits "disconnect" from the menu
+					websocket.CloseGoingAway: // when the user closes the tab
+					wsClosedByClient = true
```

Comment:
Should we stop here?

Reply:
d9d1d6f87852a508b34da1da7665c013ff785feb